### PR TITLE
Update README.md file :

### PR DESCRIPTION
**Pull request by mani87**
Update README.md file seperating
legacy code and API based code.

Closes https://github.com/fossasia/open-event-server/issues/4543

Fixes #

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Changes proposed in this pull request:

-
-
-

### CODE REVIEW

**Comment by niranjan94** [comment]
@0x48piraj thanks for your review. But a useful review would be, if you give more information instead of just saying `Work needed!`.

**Comment by niranjan94** [comment]
@mani87 also provide a proper PR title

**Comment by 0x48piraj** [comment]
@niranjan94 
Sorry, I told him the problems on gitter channel :smile:

**Comment by niranjan94** [comment]
@0x48piraj do not use gitter for code reviews. :) ... Ensure you put in your reviews in the specific PR/issue

**Comment by mani87** [comment]
@niranjan94 is it fine now? Can you please see this? 
Thankyou :)

File: README.md [hunk 1]
```diff
@@ -30,8 +30,8 @@ The **Open Event Server** exposes a well documented [JSON:API Spec](http://jsona
 ## Demo Version
 
 A demo version is automatically deployed from our repositories:
-* Deployment from the `master` branch - **[open-event-api.herokuapp.com](http://open-event-api.herokuapp.com/)**
-* Deployment from the `development` branch - **[open-event-api-dev.herokuapp.com](http://open-event-api-dev.herokuapp.com/)**
+* Deployment from the `master` branch - **[api.eventyay.com/open-event-api-dev.herokuapp.com](http://api.eventyay.com/open-event-api.herokuapp.com/)**
```
Comment (niranjan94) [inline README.md]: `api.eventyay.com/open-event-api-dev.herokuapp.com` is invalid. Two deployments happen
- api.eventyay.com
- open-event-api.herokuapp.com

File: README.md [hunk 2]
```diff
@@ -30,8 +30,8 @@ The **Open Event Server** exposes a well documented [JSON:API Spec](http://jsona
...
+* Deployment from the `development` branch - **[ api-dev.eventyay.com/open-event-api-dev.herokuapp.com](http://api-dev.eventyay.com/open-event-api-dev.herokuapp.com/)**
```
Comment (niranjan94) [inline README.md]: Same issue as above

**Comment by mani87** [comment]
Is this one fine?

**Comment by niranjan94** [comment]
@mani87 it is still incorrect. Please read my comment. https://github.com/fossasia/open-event-server/pull/4557#discussion_r154521105

_two_ different links are needed per branch

**Comment by mani87** [comment]
@niranjan94 now?
Thankyou for responding! :)

**Comment by niranjan94** [comment]
@mani87 what you have done is 

* Deployment from the `master` branch - **[api.eventyay.com](http://open-event-api.herokuapp.com/)**
* Deployment from the `development` branch - **[ api-dev.eventyay.com](http://open-event-api-dev.herokuapp.com/)**

We don't want api.eventyay.com to goto open-event-api.herokuapp.com. We need both 
api.eventyay.com and open-event-api.herokuapp.com. 

Again,  my comment https://github.com/fossasia/open-event-server/pull/4557#discussion_r154521105 

Two different deployment links per branch = a total of 4 links

**Comment by mani87** [comment]
Some efforts. Can you see @niranjan94 ?
Thank you :)

File: README.md [hunk 3]
```diff
@@ -30,8 +30,8 @@ The **Open Event Server** exposes a well documented [JSON:API Spec](http://jsona
 ## Demo Version
 
 A demo version is automatically deployed from our repositories:
-* Deployment from the `master` branch - **[open-event-api.herokuapp.com](http://open-event-api.herokuapp.com/)**
-* Deployment from the `development` branch - **[open-event-api-dev.herokuapp.com](http://open-event-api-dev.herokuapp.com/)**
+* Deployment from the `master` branch - **[api.eventyay.com][open-event-api.herokuapp.com](http://open-event-api.herokuapp.com/)**
```
Comment (niranjan94) [inline README.md]: Markdown syntax for links is incorrect. Should be like

```markdown
Deployment from the `master` branch - **[api.eventyay.com](https://api.eventyay.com) &
 [open-event-api.herokuapp.com](https://open-event-api.herokuapp.com)**
```

**Comment by mani87** [comment]
Done @niranjan94

**Comment by mani87** [comment]
@niranjan94  updated!
Thankyou :)

**Comment by mani87** [comment]
@niranjan94  can you please review?

**Pull request merged**